src/ssl/support.cc

```diff
@@ -1046,16 +1046,41 @@ findCertIssuer(Security::CertList const &list, X509 *cert)
     return false;
 }
 
+/// \return true if the issuer exist in trusted CA database
```

Comment:
Please clarify which "issuer" we are looking for and avoid assuming what the `connContext` parameter represents (unless that assumption is necessary):

```
s/issuer/`cert` issuer/
s/trusted CA database/the certificates stored in `connContext`/
```